packages/testing/src/execution_testing/test_types/block_access_list/account_changes.py

```diff
@@ -111,7 +111,7 @@ class BalAccountChange(CamelModel, RLPSerializable):
     storage_changes: List[BalStorageSlot] = Field(
         default_factory=list, description="List of storage changes"
     )
-    storage_reads: List[HexNumber] = Field(
+    storage_reads: List[ZeroPaddedHexNumber] = Field(
```

Comment:
I think `StorageKey` would be better (requires import!):

```suggestion
    storage_reads: List[StorageKey] = Field(
```
and consistent with:
 
- `BalAccountExpectation` - https://github.com/fselmo/execution-specs/blob/ab17e0642769c0c9bc4b3a4a34ff21dfa09e7d3d/packages/testing/src/execution_testing/test_types/block_access_list/expectations.py#L48-L50
- `BalAccountAbsentValues` - https://github.com/fselmo/execution-specs/blob/ab17e0642769c0c9bc4b3a4a34ff21dfa09e7d3d/packages/testing/src/execution_testing/test_types/block_access_list/account_absent_values.py#L94-L98

Reply:
`StorageKey` is exactly what we just moved out of (`Bytes32`) [here](https://github.com/ethereum/execution-specs/pull/1912/changes#diff-e154b39cdaa261fe7e49a21c81aff37f728ddfa19ae8d97fb9a0580125030df1L89-R88) and instead opted to store storage keys as `Uint256` (hence `ZeroPaddedHexNumber`). It is purposeful to _not_ have this as Bytes32.

Reply:
Here is the recent update to the EIP: https://github.com/ethereum/EIPs/commit/6c6b1e72412460d9888e47f3549b2c926cf13b19

Reply:
Aaah, thanks @fselmo.